int8 count_c
---
int8 total_r
int8 total_c
